CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ Changelog for v1.4
 ------------------
 
 * feature #841 PhpdocParamsFixer: added aligning var/type annotations (GrahamCampbell)
+* bug #950 Tokens::isLambda - fix detection near comments (keradus)
```

Comment:
you've added an entry to the 1.4 change log and you've said you plan to merge this into 2.0. That doesn't make sense to me.